Sources/PackageCollections/Model/Collection.swift

```diff
@@ -113,6 +119,14 @@ extension PackageCollectionsModel {
     }
 }
 
+extension PackageCollectionsModel {
+    /// Represents the author of a `PackageCollection`
+    public struct CollectionAuthor: Equatable, Codable {
```

Comment:
consider CollectionAuthor -> Author

Reply:
Don't want to confuse that with package author.